server/odc-service/src/main/java/com/oceanbase/odc/service/quartz/QuartzJobService.java

```diff
@@ -59,11 +59,15 @@ public class QuartzJobService {
 
     @Autowired(required = false)
     @Qualifier(value = ("defaultScheduler"))
-    private Scheduler scheduler;
+    private Scheduler defaultScheduler;
+    @Autowired(required = false)
+    @Qualifier(value = ("commonScheduler"))
+    private Scheduler commonScheduler;
```

Comment:
this modification exits a issue is that changeQuartzJob can retrieve job class name, but pause or other operation can't know what type of this job is, user should pass scheduler to method. as user has know what scheduler is, why not remote auto judge code and let scheduler as a input for function call

Reply:
i create a new QuartzJobServiceProxy to solve this problem